### PR TITLE
Service principal migration guide updates for resources and properties

### PR DESCRIPTION
**Pull request by sureshja**

### CODE REVIEW

**Comment by CelesteDG** [comment]
Hi @sureshja - Is this ready to merge?

File: concepts/migrate-azure-ad-graph-property-differences.md [hunk 1]
```diff
@@ -167,18 +167,18 @@ The Azure AD Graph ServiceEndpoint resource is renamed to endpoint in Microsoft
 
 |Azure AD Graph <br>(v1.6) property |Microsoft Graph<br> property|Comments|
 |---|---|---|
-| **applicationTemplateId** | beta  &nbsp;-&nbsp;**applicationTemplateId** <br> v1.0  &nbsp;-&nbsp; _Not yet available_ | |
+| **applicationTemplateId** | beta  &nbsp;-&nbsp;**applicationTemplateId** <br> v1.0  &nbsp;-&nbsp; **applicationTemplateId** | |
```
Comment (dkershaw10) [inline concepts/migrate-azure-ad-graph-property-differences.md:170]: If the name is the same and it's in both versions, then just remove the line.

File: concepts/migrate-azure-ad-graph-resource-differences.md [hunk 2]
```diff
@@ -20,16 +20,19 @@ If a resource is **not** shown in this list, it is already available in the [v1.
 
 |Azure AD Graph <br>(v1.6) resource |Microsoft Graph<br>resource|Comments|
 |---|---|---|
-| [AppRoleAssignment](https://docs.microsoft.com/previous-versions/azure/ad/graph/api/entity-and-complex-type-reference) | beta&nbsp;-&nbsp;[appRoleAssignment](/graph/api/resources/approleassignment?view=graph-rest-beta)<br>v1.0 - _Not yet available_ | |
+| [Application](https://docs.microsoft.com/previous-versions/azure/ad/graph/api/entity-and-complex-type-reference) | beta - [application](/graph/api/resources/application?view=graph-rest-beta)<br>v1.0 - [application](/graph/api/resources/application?view=graph-rest-1.0) ||
```
Comment (dkershaw10) [inline concepts/migrate-azure-ad-graph-resource-differences.md:23]: Again - this is all wrong.
Lines should only be present IF there's any difference in the resource name between AAD Graph and Microsoft Graph.

File: concepts/migrate-azure-ad-graph-property-differences.md [hunk 3]
```diff
@@ -167,18 +167,18 @@ The Azure AD Graph ServiceEndpoint resource is renamed to endpoint in Microsoft
 
 |Azure AD Graph <br>(v1.6) property |Microsoft Graph<br> property|Comments|
```
Comment (dkershaw10) [inline concepts/migrate-azure-ad-graph-property-differences.md:168]: You also need to update the user and application tables.

File: concepts/migrate-azure-ad-graph-property-differences.md (hunk 1 above)
Comment (sureshja) [reply concepts/migrate-azure-ad-graph-property-differences.md:170]: Should I not document even if there is case difference (e.g. Application vs application)?

Comment (sureshja) [reply concepts/migrate-azure-ad-graph-property-differences.md]: Just touched base with Dan. Addressed these comments.

File: concepts/migrate-azure-ad-graph-resource-differences.md (hunk 2 above)
Comment (sureshja) [reply concepts/migrate-azure-ad-graph-resource-differences.md]: Addressed.

File: concepts/migrate-azure-ad-graph-property-differences.md (hunk 3 above)
Comment (sureshja) [reply concepts/migrate-azure-ad-graph-property-differences.md:167]: Done.

File: concepts/migrate-azure-ad-graph-resource-differences.md [hunk 4]
```diff
@@ -20,16 +20,14 @@ If a resource is **not** shown in this list, it is already available in the [v1.
...
-| [ServiceEndpoint](https://docs.microsoft.com/previous-versions/azure/ad/graph/api/entity-and-complex-type-reference) | beta - [endpoint](/graph/api/resources/endpoint?view=graph-rest-beta) <br> v1.0 - endpoint _Not yet available_ | [endpoint](/graph/api/resources/endpoint?view=graph-rest-beta) is only available as part of the [group](/graph/api/resources/group?view=graph-rest-beta) resource.|
+| [ServiceEndpoint](https://docs.microsoft.com/previous-versions/azure/ad/graph/api/entity-and-complex-type-reference) | beta - [endpoint](/graph/api/resources/endpoint?view=graph-rest-beta) <br> v1.0 - [endpoint](/graph/api/resources/endpoint?view=graph-rest-1.0) | [endpoint](/graph/api/resources/endpoint?view=graph-rest-beta) is only available as part of the [group](/graph/api/resources/group?view=graph-rest-beta) resource.|
```
Comment (dkershaw10) [inline concepts/migrate-azure-ad-graph-resource-differences.md:30]: The Note should be removed.

**Pull request merged**